### PR TITLE
Add user profile

### PR DESCRIPTION
**Pull request by grigiv**

### CODE REVIEW

File: src/main/java/com/epam/service/PersonDetailsServiceImpl.java [hunk 1]
```diff
@@ -65,4 +65,11 @@ public boolean updatePassword(String email, String password) {
         if (result < 1) return false;
         return true;
     }
+
+    public boolean updatePasswordById(Integer id, String password) {
+        int result = personService.updatePasswordById(id, password);
```
Comment (ghost) [inline src/main/java/com/epam/service/PersonDetailsServiceImpl.java:70]: it would be great to check, if person with such id exists before updating

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
seems good, but how can we test it? When i went to this urls, i see 404 error.

**Review by ghost — COMMENTED** [review]
fyi

**Comment by ghost** [comment]
seems really good! Little fix please: when i click to the our logo, i want to return to the main page, but now it is not, please fix it

**Comment by grigiv** [comment]
How to check:
1. Log in as ADMIN.
2. Go to http://localhost:8080/clients/ {clinetId} (because there is no Clients list page at the moment).
For example: 
http://localhost:8080/clients/1
http://localhost:8080/clients/2
3. You can see user info (email, role, list of reservations) and you can change password

**Review by goponenkoev — APPROVED** [review]

**Review by MaltsevaKsenia — APPROVED** [review]

**Pull request merged**